src/commitment_scheme/kzg10/proof.rs

```diff
@@ -0,0 +1,111 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) DUSK NETWORK. All rights reserved.
+
+use super::Commitment;
+use dusk_bls12_381::BlsScalar;
+
+#[derive(Copy, Clone, Debug)]
+/// Proof that a polynomial `p` was correctly evaluated at a point `z`
+/// producing the evaluated point p(z).
+pub(crate) struct Proof {
```

Comment:
It is unclear why we have the `(crate)` scope directive here considering this is a public structure. Is this the struct that is exported? If no, we should also consider renaming it to avoid mismatch with the actual proof.

Reply:
This is not the `Proof` that is verified in the public API but the one used inside the commitment scheme. If you want a rename, file an issue for it, but will not change in this release IMO.

Reply:
https://github.com/dusk-network/plonk/issues/525